build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -1384,8 +1409,8 @@ CONFIG_ENABLE_ENV_GET=y
 CONFIG_ENABLE_ENV_SET=y
 CONFIG_ENABLE_ENV_UNSET=y
 CONFIG_ENABLE_FREE=y
-# CONFIG_ENABLE_IRQINFO is not set
 CONFIG_ENABLE_KILL=y
+CONFIG_ENABLE_KILLALL=y
```

Comment:
can we keep it disable?
because kill command is already enabled.

Reply:
`kill` and `killall` have different functionality, so I think we can enable both.